src/Tasks/Task-3 (Redux)/reducers/StateChange.js

```diff
@@ -0,0 +1,20 @@
+const initialState =0;
+
+const StateChange = (state=initialState , action) => {
+    switch (action.type){
+        case "INCREMENT":
+            return state + action.payload;
+        case "DECREMENT":
+            if (state>0){
+                return state - action.payload;
+            }
+            else {
+                alert ("Number smaller than 0 is not allow");
```

Comment:
You can make one initial state at the top for error and make it true for handling errors